README.md

```diff
@@ -45,34 +45,29 @@ See public/licenses.txt
 * [Docker](https://docs.docker.com/install/)
 * [Docker Compose](https://docs.docker.com/compose/install/)
 * (For shapefile support) access to an [ogre](http://ogre.adc4gis.com) server
-* (Optional) For automatic spatial and temporal extraction from the search text, clone and set up an [EDSC-NLP](https://git.earthdata.nasa.gov/projects/EDSC/repos/edsc-nlp/browse) server
+* (Optional) For automatic spatial and temporal entity extraction from the search text, clone and set up an [EDSC-NLP](https://git.earthdata.nasa.gov/projects/EDSC/repos/edsc-nlp/browse) server
 
-### Initial setup
-
-Run
+### Application Configuration
 
-    bin/setup
+Use the templates at `config/application.yml.example` and `config/database.yml.example` to create `config/application.yml` and `config/database.yml` respectively. Update values as necessary. Note that the Earthdata Login (URS) Configuration is optional however without the Earthdata Login configuration, Earthdata Search's functionality will be limited. If you would like to set up Earthdata Login login, you will need to perform the following steps:
 
-Note: This command will take a long time to run
-
-### Application configuration
+Register an account on [the Earthdata Login home page](https://urs.earthdata.nasa.gov/home).
```

Comment:
I think that the steps outlined to set up Earthdata Login should be a list (bullet or numbered, doesn't really matter to me.) But a list would read more clearly IMO.